src/accessControl/index.js

```diff
@@ -36,15 +36,19 @@ export function hasAccess(resourceInfo) {
   return hasAccessibleAcl(resourceInfo) || acp.hasLinkedAcr(resourceInfo);
 }
 
-export function isAcp(resourceUrl, fetch) {
-  return resourceUrl && acp3.isAcpControlled(resourceUrl, { fetch });
+export async function isAcp(resourceUrl, fetch) {
+  const isAcpControlledResource = await acp3.isAcpControlled(resourceUrl, {
+    fetch,
+  });
+  return resourceUrl && isAcpControlledResource;
 }
```

Comment:
```suggestion
export async function isAcp(resourceUrl, fetch) {
  return resourceUrl && await acp3.isAcpControlled(resourceUrl, {
    fetch,
  });
}
```
What about this, without the intermediary variable ?